vault/datadog_checks/vault/data/conf.yaml.example

```diff
@@ -45,10 +45,22 @@ init_config:
 #
 instances:
 
+    ## @param use_openmetrics - boolean - optional - default: false
+    ## Use the latest OpenMetrics implementation for more features, better performance,
+    ## and more appropriate metric types by default. If running on Agent v6, you must
```

Comment:
```suggestion
    ## and more metric types by default. If running on Agent v6, you must
```

Reply:
This file is auto-generated, see https://github.com/DataDog/integrations-core/pull/11293#discussion_r799036066